carina-utils/src/main/java/com/qaprosoft/carina/core/foundation/report/ReportContext.java

```diff
@@ -663,6 +663,7 @@ public static String getCucumberReportLink() {
             }
             link = String.format("%s/%d/%s/%s/%s/feature-overview.html", report_url, rootID, ARTIFACTS_FOLDER, folder, subFolder);
         } else {
+            //TODO: link might be broken on CI like https://github.com/qaprosoft/carina/issues/1001
```

Comment:
![INFO](https://sonarsource.github.io/sonar-github/severity-info.png 'Severity: INFO') Complete the task associated to this TODO comment. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](http://demo.qaprosoft.com/sonarqube/coding_rules#rule_key=squid%3AS1135)